### ISSUE TITLE
Bold and italic markup

### ISSUE THREAD

**Issue by lorenaramonda** [MarvinRudolph/storyblok-rich-text-renderer#14]
Hello @MarvinRudolph,
could we use <strong> and <em> instead of <b> and <i> as markup for bold and italic? 😬 
I can also send a PR by myself if you agree.

Let me know,
thanks!

**Comment by phaberest** [comment]
I agree with you but it depends the way you the content is written. I might be a good idea to have a setting for such a thing.

Technically `<b>` and `<i>` are just aesthetics, while `<strong>` and `<em>` are more important  if, let's say, someone is using a screen reader to parse your website. (https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em)

**Comment by lorenaramonda** [comment]
I agree. A setting could suit for everyone. In most cases one would choose semantic `<strong>` and `<em>`, but there could be cases where one preferes `<b>` and `<i>`. So giving the flexibility to choose is a good solution.
